tensorboard/compat/tensorflow_stub/io/gfile.py

```diff
@@ -158,7 +157,7 @@ def _write(self, filename, file_content, mode):
 
     def glob(self, filename):
         """Returns a list of files that match the given pattern(s)."""
-        if isinstance(filename, six.string_types):
+        if isinstance(filename, (str,)):
```

Comment:
It looks like in every\* place that we had used `six.string_types` we're just passing it into isinstance(), so it seems better to just replace it with `str` rather than `(str,)` since the latter is entirely vestigial?

\* The one exception being in a docstring, but in that case I think `str` would also be more intelligible than `(str,)`

Reply:
Good point. Done.